scripts/sct_deepseg_sc.py

```diff
@@ -136,7 +136,7 @@ def crop_image_around_centerline(filename_in, filename_ctr, filename_out, crop_s
     x_lst, y_lst = [], []
     data_im_new = np.zeros((crop_size, crop_size, im_in.dim[2]))
     for zz in range(im_in.dim[2]):
-        if 1 in np.array(data_ctr[:, :, zz]):
+        if np.sum(np.array(data_ctr[:, :, zz])):
```

Comment:
could also use `np.any` for more semantic

Reply:
done